dadi/lib/handlers/image.js

```diff
@@ -382,11 +384,18 @@ ImageHandler.prototype.convert = function (stream, imageInfo) {
                 (width / height) !== (imageInfo.width / imageInfo.height) &&
                 !self.storageHandler.notFound
               ) {
+                console.log({
+                  left: crops.x1,
+                  top: crops.y1,
+                  width: crops.x2 - crops.x1,
+                  height: crops.y2 - crops.y1
+                })
+
                 sharpImage.extract({
                   left: crops.x1,
                   top: crops.y1,
-                  width: crops.x1 + crops.x2,
-                  height: crops.y1 + crops.y2
+                  width: crops.x2 - crops.x1,
```

Comment:
@eduardoboucas I'm concerned about this change to this part of resizing - do you think this calculation change was required because the `extract` method works differently to how we did it with lwip?